src/systems/hubs-systems.js

```diff
@@ -6,9 +6,28 @@ import { HoverMenuSystem } from "./hover-menu-system";
 import { SuperSpawnerSystem } from "./super-spawner-system";
 import { HapticFeedbackSystem } from "./haptic-feedback-system";
 import { SoundEffectsSystem } from "./sound-effects-system";
+import { proxiedUrlFor } from "../utils/media-utils";
+import { load, prepareForRender } from "../utils/preload";
+import cameraModelSrc from "../assets/camera_tool.glb";
+
+async function preloadPenAndCamera(sceneEl) {
+  const objects = [];
+
+  const pen = await load(
+    proxiedUrlFor("https://asset-bundles-prod.reticulum.io/interactables/DrawingPen/DrawingPen-34fb4aee27.gltf")
+  );
+  pen.traverse(o => objects.push(o));
+
+  const camera = await load(cameraModelSrc);
```

Comment:
Note that the camera-tool currently does not load models through the GLTF cache, so I don't think this preloading will actually help it very much, since it will have its own instances of geometry/textures